### PR TITLE
infra: Enable the workflow that prepares and tags a release commit

### PR DESCRIPTION
**Pull request by poncovka**
Automated releases should be disabled for branched Fedoras, but it should be still possible to manually trigger the workflow that creates a release commit via GitHub.

### CODE REVIEW

File: .github/workflows/release-periodically.yml.j2 [hunk 1]
```diff
@@ -1,4 +1,3 @@
-{% if distro_release == "rawhide" %}
 name: Release periodically
 # Create a GitHub release on schedule
```
Comment (VladimirSlavik) [inline .github/workflows/release-periodically.yml.j2:2]: Maybe please add a comment that it's kept on the fedora branches with only manual trigger to make things easier. The comment is no longer the whole truth :(

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

**Comment by jkonecny12** [comment]
I restarted the cockpit-ci test, however, if they went red again please rebase this PR to fix them.

**Pull request merged**